go/vt/topo/test/vschema.go

```diff
@@ -62,4 +69,13 @@ func CheckVSchema(ctx context.Context, t *testing.T, ts topo.Impl) {
 	if err == nil || !strings.HasPrefix(err.Error(), want) {
 		t.Errorf("SaveVSchema: %v, must start with %s", err, want)
 	}
+
+	// test GetShardNames
```

Comment:
Maybe a comment of why we do this, something like:

make sure the vschema as stored in the topology server is not returned as a shard name, as usually they share the same directory location.